webapp/cas/go.mod

```diff
@@ -13,3 +13,37 @@ require (
 	go.dedis.ch/onet/v3 v3.2.10
 	go.dedis.ch/protobuf v1.0.11
 )
+
+require (
```

Comment:
Lazy question: is this the new `go.mod` format with two `require` sections? One for the requirements, and one for the indirect ones?

Reply:
Dunno, golang 1.17 forces me to add the indirect deps and generated this format. But moving indirect deps to direct doesn't makes `go mod tidy` reorder theses.